addons/events/src/components/Event.tsx

```diff
@@ -17,7 +17,7 @@ interface StyledTextareaProps {
 
 const StyledTextarea = styled(({ shown, failed, ...rest }: StyledTextareaProps) => (
   <Textarea {...rest} />
-))(
+))<{ shown: boolean; failed: boolean }>(
```

Comment:
what does this do?

Reply:
Adds types to the function that reduces theme+props into a style-object